tools/cpplint.py

```diff
@@ -526,6 +525,8 @@
 # Match string that indicates we're working on a Linux Kernel file.
 _SEARCH_KERNEL_FILE = re.compile(r'\b(?:LINT_KERNEL_FILE)')
 
+_NULL_TOKEN_PATTERN = re.compile(r'\sNULL\s?')
```

Comment:
How about `\bNULL\b`?

Reply:
Yeah, using a word boundary would be better. I've updated now. Thanks